lib/active_merchant/billing/credit_card_methods.rb

```diff
@@ -104,7 +104,7 @@ def valid_start_year?(year)
       # master:           http://www.mastercard.com/ca/merchant/en/getstarted/Anatomy_MasterCard.html
       # jcb:              http://www.jcbcard.com/security/info.html
       # diners_club:      http://www.dinersclub.com/assets/DinersClub_card_ID_features.pdf
-      # discover:         https://www.discover.com/credit-cards/help-center/glossary.html
+      # discover:         https://www.discoverglobalnetwork.com/downloads/IPP_VAR_Compliance.pdf
```

Comment:
@prashcr The card number length change looks good, will you rollback the change to the URL?